### PR TITLE
Fixes QueryHelpers.AddQueryString() throws when one of the dictionary values is null

### PR DESCRIPTION
**Pull request by artdedeco**
- Skips appending parameters which are null.
- Add some missing docs

Addresses #16647

### CODE REVIEW

File: src/Http/WebUtilities/src/QueryHelpers.cs [hunk 1]
```diff
@@ -91,6 +99,9 @@ public static string AddQueryString(string uri, IDictionary<string, string> quer
             sb.Append(uriToBeAppended);
             foreach (var parameter in queryString)
             {
+                if (parameter.Value == null)
```
Comment (Tratcher) [inline src/Http/WebUtilities/src/QueryHelpers.cs]: Style guide: always use braces `{ }`, even for single line ifs.

**Review by Tratcher — APPROVED** [review]
One style issue and this should be good to go.

**Comment by Tratcher** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=430736

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/3320
https://github.com/aspnet/AspNetCore-Internal/issues/2730

**Comment by Tratcher** [comment]
Thanks

**Comment by Schoof-T** [comment]
@Tratcher I'm still getting this error, but to be able to use this I have to use the Nuget https://www.nuget.org/packages/Microsoft.AspNetCore.WebUtilities which hasn't been updated since 2018. Can you use the QueryHelpers without this nuget (in a Class Library), and how?

**Pull request merged**